daceml/onnx/nodes/codegen.py

```diff
@@ -327,7 +327,7 @@ def expand_node(node, state, sdfg):
     inputs_on_host = [True for _ in range(len(inputs))]
 
     actual_node_schedule = node.schedule
-    if node.schedule == dtypes.ScheduleType.CPU_Multicore or node.schedule == dtypes.ScheduleType.Default:
+    if node.schedule == dtypes.ScheduleType.CPU_Multicore or node.schedule == dtypes.ScheduleType.Default or node.schedule == dtypes.ScheduleType.Sequential:
```

Comment:
Not sure if this is a good idea. What happens if an ORT expansion is called within a OpenMP parallel map?

Reply:
Yep, that was the case